crates/frontend/src/api.rs

```diff
@@ -194,3 +194,24 @@ pub async fn fetch_experiment(
         Err(e) => Err(ServerFnError::ServerError(e.to_string())),
     }
 }
+
+pub async fn fetch_types(
+    tenant: String,
+    page: i64,
+    count: i64,
+) -> Result<FetchCustomTypeResponse, ServerFnError> {
+    let client = reqwest::Client::new();
+    let host = use_host_server();
+
+    let url = format!("{host}/types/{page}/{count}");
```

Comment:
Won't it be better to use `request` function for this?

Reply:
Good point, I'll add this